third_party/validator/project_organization_policy.go

```diff
@@ -0,0 +1,126 @@
+package google
```

Comment:
Make sure to add this file to https://github.com/GoogleCloudPlatform/magic-modules/blob/master/provider/terraform_object_library.rb#L76

Reply:
Added path to org policy file